public_suffix_list.dat

```diff
@@ -11877,6 +11877,16 @@ wmflabs.org
 // Submitted by Stefano Rivera <stefano@yola.com>
 yolasite.com
 
+// Yombo : https://yombo.net
+// Submitted by Mitch Schwenk <mitch@yombo.net>
+homelink.one
+ybo.faith
+ybo.party
+ybo.review
+ybo.science
+ybo.trade
+yombo.me
+
```

Comment:
Please review https://github.com/publicsuffix/list/wiki/Guidelines#prepare-the-change for how to properly order these.